source/test/TestApp/PlayFabApiTest.cpp

```diff
@@ -730,6 +767,43 @@ namespace PlayFabUnit
         }
     }
 
+     // GROUPS API
+    void PlayFabApiTest::GroupsApiTest(TestContext& testContext)
```

Comment:
This new set of tests should be a whole new test file: PlayFabGroupsTest.cpp/.h

I would have many more comments and changes to suggest if we don't do that and leave it here. But I think if we separate this to a separate file & test suite, we'd negate those other comments, so I'll just make this suggestion alone.